View/ViewHandler.php

```diff
@@ -448,6 +449,18 @@ private function initResponse(View $view, $format)
             if ($serializer instanceof SerializerInterface) {
                 $context = $this->getSerializationContext($view);
                 $content = $serializer->serialize($data, $format, $context);
+            //       test for symfony 2.7 version 
+            } elseif (class_exists('\Symfony\Component\Serializer\Normalizer\ObjectNormalizer') && $serializer instanceof SymfonySerializerInterface) {
+                    
+                $context = $this->getSerializationContext($view);
```

Comment:
If I create a new context class that implements the jms serializer context methods, would it be ok for bc breaks ?